source/MRViewer/MRSpaceMouseHandlerHidapi.h

```diff
@@ -13,11 +12,14 @@
 #ifdef __APPLE__
 #include <hidapi/hidapi_darwin.h>
 #endif
-
+#ifdef _WIN32
+#include <windows.h>
+#include <hidapi/hidapi_winapi.h>
+#endif
 
 namespace MR
 {
-class SpaceMouseHandlerHidapi : public SpaceMouseHandler
+class SpaceMouseHandlerHidapi : public SpaceMouseHandler, public MultiListener<PostFocusListener>
```

Comment:
if it listens only PostFocusListener mb inherit it from PostFocusListener directly?